packages/styled-ui-docs/pages/drawer.mdx

```diff
@@ -1,96 +1,308 @@
+            disabled={!closeOnEsc && !closeOnOutsideClick}
+            onChange={toggleIsCloseButtonVisible}
+          />
+          <Space width="2x" />
+          <Text fontFamily="mono" whiteSpace="nowrap">isCloseButtonVisible</Text>
+        </TextLabel>
+      </FormGroup>
+      <FormGroup>
+        <TextLabel display="flex" alignItems="center">
+          <Checkbox
+            checked={closeOnEsc}
+            disabled={!isCloseButtonVisible && !closeOnOutsideClick}
+            onChange={toggleCloseOnEsc}
+          />
+          <Space width="2x" />
+          <Text fontFamily="mono" whiteSpace="nowrap">closeOnEsc</Text>
+        </TextLabel>
+      </FormGroup>
+      <FormGroup>
+        <TextLabel display="flex" alignItems="center">
+          <Checkbox
+            checked={closeOnOutsideClick}
+            disabled={(!isCloseButtonVisible && !closeOnEsc) || !backdrop}
+            onChange={toggleCloseOnOutsideClick}
+          />
+          <Space width="2x" />
+          <Text fontFamily="mono" whiteSpace="nowrap">closeOnOutsideClick</Text>
+        </TextLabel>
+      </FormGroup>
+      <FormGroup>
+        <TextLabel display="flex" alignItems="center">
+          <Checkbox
+            checked={isOverlayVisible}
+            disabled={!backdrop}
+            onChange={toggleIsOverlayVisible}
+          />
+          <Space width="2x" />
+          <Text fontFamily="mono" whiteSpace="nowrap">DrawerOverlay</Text>
+        </TextLabel>
+      </FormGroup>
+      <FormGroup>
+        <TextLabel display="flex" alignItems="center">
+          <Checkbox checked={isHeaderVisible} onChange={toggleIsHeaderVisible} />
+          <Space width="2x" />
+          <Text fontFamily="mono" whiteSpace="nowrap">DrawerHeader</Text>
+        </TextLabel>
+      </FormGroup>
+      <FormGroup>
+        <TextLabel display="flex" alignItems="center">
+          <Checkbox checked={isBodyVisible} onChange={toggleIsBodyVisible} />
+          <Space width="2x" />
+          <Text fontFamily="mono" whiteSpace="nowrap">DrawerBody</Text>
+        </TextLabel>
+      </FormGroup>
+      <FormGroup>
+        <TextLabel display="flex" alignItems="center">
+          <Checkbox checked={isFooterVisible} onChange={toggleIsFooterVisible} />
+          <Space width="2x" />
+          <Text fontFamily="mono" whiteSpace="nowrap">DrawerFooter</Text>
+        </TextLabel>
+      </FormGroup>
+      <Slide
+        in={isOpen}
+        from={placement}
+        finalHeight="100vh"
+      >
+        {styles => (
+          <Drawer
+            autoFocus={autoFocus}
+            backdrop={backdrop}
+            isCloseButtonVisible={isCloseButtonVisible}
+            isOpen={isOpen}
+            closeOnEsc={closeOnEsc}
+            closeOnOutsideClick={closeOnOutsideClick}
+            onClose={onClose}
+            placement={placement}
+            size={size}
+          >
+            {isOverlayVisible && (
+              <DrawerOverlay />
```

Comment:
```jsx
<DrawerOverlay opacity={styles.opacity} />
```

Reply:
Fixed in bd9d1faa84f2591d14982c6f3875cbacf1740a75 and 7c9fb92f249cce05ba7d239c6ffaa7837408d158